app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -138,6 +155,39 @@ class ProfileResetPinActivityTest {
     }
   }
 
+  @Test
+  fun testProfileResetPinActivity_startActivityWithAdmin_inputPin_inputConfirmPin_pressImeActionButton_checkReturnsToProfileEditActivity() { // ktlint-disable max-line-length
```

Comment:
Update this to `fun testResetPin_startWithAdmin_inputPinAndConfirmPin_pressImeAction_checkReturnsToProfileEdit() {` so that you can remove the comment.

Reply:
Done